src/modal/vlmodal.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+/**
+ * @ngdoc directive
+ * @name vlui.directive:vlModal
+ * @description
+ * # vlModal
+ */
+angular.module('vlui')
+  .directive('vlModal', function (VlModals) {
```

Comment:
This is a good question for you and @kanitw actually, @domoritz — I did not use `<modal>` here because that is a common name and it would preclude us or others using this directive inside another app that defined its own `<modal>`, and because I had seen `<vl-plot-group>` and other vl-prefixed directives as prior art.

If we do not need this to interoperate with other angular UI code, and other potential app environments, I'm fine renaming it to `<modal>`—but there's also an argument to be made to keep the `vl-` prefixing for consistency. Let me know which way you'd like to go.